USER_GUIDE.md

```diff
@@ -119,7 +120,39 @@ response = client.indices.delete(
 print('\nDeleting index:')
 print(response)
 ```
+## Making API Calls
+
+### Point in Time API calls
```

Comment:
Should we give a brief about Point In Time here?

Reply:
Will update it with the official documentation later